cimi-deployment/docker-compose.yml

```diff
@@ -30,27 +31,27 @@ services:
     ports:
      - "5601:5601"
   rc:
-    image: sixsq/ring-container:${SLIPSTREAM_VERSION}
+    image: sixsq/ring-container:3.54-SNAPSHOT
```

Comment:
Was this hardcoded intentionally?  (Same elsewhere in file.)

Reply:
changed to "latest"